Simplenote/Classes/SPTitleView.swift

```diff
@@ -0,0 +1,17 @@
+//
+//  SPTitleView.swift
+//  Simplenote
+//
+//  Created by Dan Roundhill on 10/2/17.
+//  Copyright © 2017 Automattic. All rights reserved.
+//
+
+import UIKit
+
+class SPTitleView: UIView {
+
+    override var intrinsicContentSize: CGSize {
```

Comment:
This would be SO MUCH EASIER with Autolayout!! One of these days we should drop all of the manual layout code here. Badly